src/frontend/src/elements/animations/LetterHightlight.tsx

```diff
@@ -0,0 +1,26 @@
+// import { PropsWithChildren } from "react";
+import "./style.css";
```

Comment:
```suggestion
import "./letter-highlight.css";
```
bitte file umbenennen